CONTRIBUTING.md

```diff
@@ -35,6 +35,19 @@ If you choose to write some code, we have some conventions as follows.
 
 ## Guidelines
 
+### Setup Optuna
+
+First of all, fork Optuna on GitHub.
+You can learn about fork in the official [documentation](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).
+
+After forking, download and install Optuna on your computer.
+
+```bash
+git clone git@github.com:your_name/optuna.git
```

Comment:
How about capitalizing it to make it obvious that it is a variable?
```suggestion
git clone git@github.com:YOUR_NAME/optuna.git
```

Reply:
Thank you for the suggestion. I updated. 📝